src/Controller/DBHandler.js

```diff
@@ -0,0 +1,450 @@
+import { get, ref, set } from "firebase/database";
+import { database } from "../firebase";
+import axios from "axios";
+
+export default class DBHandler {
+  constructor(uid, setErrorMessage, setGoHome) {
+    this.uid = uid;
+    this.setErrorMessage = setErrorMessage;
+    this.setGoHome = setGoHome;
+  }
+
+  //check the deck valid for user
+  //No return
+  checkUserDeckID = async (deckID, isGoHome) => {
```

Comment:
I think we could find a better name than `isGoHome` here as it isn't rly clear what that means